dynatrace/src/test/java/com/dynatrace/opentelemetry/metric/DynatraceMetricExporterTest.java

```diff
@@ -167,10 +167,10 @@ public void addDefaultDimensions() throws IOException {
   }
 
   @Test
-  public void testWithLabels() throws IOException {
+  public void testWithAttributes() throws IOException {
```

Comment:
Nit: some of the test have the `public` modifier while others don't. It is not necessary to have the `public` modifier starting from JUnit 5. Could remove it for a more uniform look. :)

Reply:
Done. :)